src/pages/contributors/herve-le-meur.adoc

```diff
@@ -0,0 +1,78 @@
+= Hervé Le Meur
+:page-name: Hervé Le Meur
+:page-linkedin: 
+:page-twitter: 
+:page-github: hlemeur
+:page-email: 
+:page-image: avatar/herve-le-meur.jpg
+:page-pronouns: He/Him/His
+:page-location: Paris, France
+:page-firstcommit: 2020
+:page-datepublished: 2024-04-10
+:page-featured: true
+:page-intro: Hervé Le Meur is a software architect and current member of the Jenkins Infrastructure team. He was introduced to the project via Jenkins-x and then moved on to work on Jenkins as part of the Infrastructure team.
+
+Despite coming from a more analog background, Hervé got his first real taste for technology with the Amstrad CPC 464 computer.
+Today, when he's not working on Jenkins tasks, Hervé likes to go for walks outside with his family, catch up on reading, and unwind watching his favorite shows.
+
+== What is your background prior to contributing to Jenkins? 
+
+After graduating university, I worked at a company for 10 years, but there was no CI/CD there.
+I did not have an opportunity to use open-source products there, but my next job had teams using Jenkins.
+I started befriending developers there, worked to become a software architect, and spent half a year integrating CD.
+Through my work, I learned more about Kubernetes & CI/CD, and in doing so I contributed quite a lot to Jenkins-x, which is what we used as the CI/CD tool.
+However, due to the project being in beta, it was more complicated than I anticipated.
```

Comment:
```suggestion
As Jenkins X was fresh from the oven and as I was new to Kubernetes, this was a much more difficult task than anticipated, especially since the passage of Jenkins X to beta, making me redo most of what I put in place several times.
Through this work, I learned a lot about Kubernetes & CI/CD, and in doing so I contributed quite a lot to Jenkins X, my first contributions to the open-source world.
```